keyboards/cutie_club/novus/readme.md

```diff
@@ -0,0 +1,19 @@
+# novus
+
+![novus](https://i.imgur.com/kuCDFb0.png)
+
+Novus 60% Keyboard
+
+* Keyboard Maintainer: [Cutie Club](https://github.com/cutie-club)
+* Hardware Supported: Atmega32u4 based pcb for Novus keyboard
+* Hardware Availability: https://mechsandco.com/products/gb-novus60-keyboard
+
+Make example for this keyboard (after setting up your build environment):
```

Comment:
Could you add instructions on how to enter the bootloader, to flash new firmware? 

Eg: 
```markup
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `RESET` if it is available
```

Reply:
THankyou good plan